plugin/src/main/java/com/playmonumenta/scriptedquests/managers/NpcTradeManager.java

```diff
@@ -147,12 +147,13 @@ public void trade(Plugin plugin, Villager villager, Player player, PlayerInterac
 						}
 						lockedSlots.append(i);
 						modified = true;
-						continue;
+						continue recipes;
```

Comment:
What's this? I haven't seen this syntax before.

Reply:
break/continue with a label is used to exit more than one loop. Here it will jump to the loop labelled `recipes`.